@here/harp-mapview/lib/ClipPlanesEvaluator.ts

```diff
@@ -877,7 +897,7 @@ export class TiltViewClipPlanesEvaluator extends TopViewClipPlanesEvaluator {
             // Step-wise calculate angle between camera eye vector and tangent
 
             // Calculate angle between surface normal(below camera position) and tangent.
-            const alpha = Math.asin(r / d);
```

Comment:
This is also completely wrong, the r & d values are othogonal to each other, neither is the hypotenuse, hence `atan` is the only valid way to calculate the angle.

Reply:
that's not right, see my comment above.